### ISSUE TITLE
Old deployments are indexed

### ISSUE THREAD

**Issue by gregdorward** [redbadger/website-honestly#470]
**Description of the issue:**
Raised by Stanton with the following description:

"FYI just found that this deployment of the website is indexed: http://ec2-54-247-187-35.eu-west-1.compute.amazonaws.com/

not sure where thats from, but think it needs a robots.txt or shutting down so we don’t get penalised for dupe content / show old members of staff etc."

We will need to look into this one as a spike to see what impact a robots.txt might have on the site, and also whether we need the site to be indexed to support the old pages we still use

**Comment by Xanndarr** [comment]
It's luckily not all deployments. There was a stray server running from **2013** at that address. I've stopped it, so issue should now be resolved. Visit http://54.247.187.35/ or http://ec2-54-247-187-35.eu-west-1.compute.amazonaws.com/ in private browsing (to avoid caching) and website should no longer be reachable.

**Comment by Xanndarr** [comment]
Scratch that, our current tech page is being served from this address. We can pull down this instance once the tech page MVP is out

**Comment by JoeStanton** [comment]
`robots.txt`?

**Comment by chris-brookes** [comment]
@Xanndarr - Is it just for the current tech page, or is it for the old case study pages too? eg. https://red-badger.com/our-work/case-study/camden-market/

**Comment by dpiatek** [comment]
As we no longer need to link to the old website, I've retired the instance that was serving this content from a public ip. I've updated the proxy to not redirect to it either. I've checked our GA account and the content indexed by Google was not receiving any significant traffic - or is actually relevant to the current site (tech pages, case studies) as content has been migrated (as far as I can tell from what was redirected).

Removing from the search results is trickier as this is an amazon aws domain (and was already serving a dis-allowing robots.txt) ie. we can't just remove urls in Google webmaster tools. I've considered keeping the instance and adding and rewrite rule on it but it just getting deployments working on it would quite a bit of effort, so decided to go with what seems the most cost and time effective route.

As a side this seems to be quite a common issue: google `site:compute.amazonaws.com`
